### DISCUSSION TITLE
html->pdf, how can I avoid pandoc modifying the styling?

### DISCUSSION THREAD

**Discussion by elondres-mim** [jgm/pandoc#9567] [Q&A]
I have HTML files I'd like to use pandoc to convert to pdf format. Using pagedjs-cli or wkhtmltopdf directly comes out great, using weasyprint directly comes out ok. Using any three of these output options with pandoc results in much of the source document's styling being removed and replaced with different styling. 

Is it possible to configure my invocation of pandoc to more closely mirror pagedjs-cli and wkhtmltopdf's default options?

**Comment by jgm** [comment]
If your source files are HTML, just use pagedjs-cli directly.  Pandoc preserves structure but does not always preserve stylistic information. If you must use pandoc, try using `--css` and passing in a file with whatever CSS styles the source document uses. Note that CSS styles defined in the document itself using a `<style>` element will just be ignored.
